x-pack/plugins/ml/public/application/contexts/ml/ml_context.ts

```diff
@@ -12,17 +12,17 @@ import { MlServicesContext } from '../../app';
 
 export interface MlContextValue {
   combinedQuery: any;
-  currentIndexPattern: DataView; // TODO this should be IndexPattern or null
+  currentDataView: DataView; // TODO this should be IndexPattern or null
```

Comment:
Nit - could change the `IndexPattern` in the comment to `DataView`

Reply:
Updated in https://github.com/elastic/kibana/pull/116455/commits/7ee9241872df9cecf890ae7dce916bfa3f491d84